controllers/send-widget.controller.es6

```diff
@@ -263,16 +265,16 @@ export default class SendWidgetController {
       .call()
       .then(account => {
         // Check if sending this transaction would make balance go below minimum balance
-        let minimumBalance = 20 + (account.subentry_count) * 10;
+        let minimumBalance = 2 * BASE_RESERVE + (account.subentry_count) * BASE_RESERVE;
```

Comment:
What do you think about having a constant like `const STARTING_BALANCE = 2 * BASE_RESERVE`?  
Then you can use `STARTING_BALANCE` in place of `2 * BASE_RESERVE` and in the future you only have to make one change, if there is any change to how minimum balance is calculated?
Apart from that, looks good.

Reply:
Good idea but I don't think it (how starting balance is calculated) will change any time soon so leaving it as it is for now.